preprocessing/berlin/compile-berlin-geojson.js

```diff
@@ -11,6 +11,17 @@ var opening_hours = require('opening_hours');
 var INPUT_FILE = "./preprocessing/berlin/raw/markets-berlin.json";
 var OUTPUT_FILE = "./cities/berlin.json";
 
+// Sort input by title
```

Comment:
You could wrap the code block in a function which has a descriptive title which lets the comment unneeded.